docs/netlify.toml

```diff
@@ -0,0 +1,13 @@
+[build]
+publish = "public"
+command = "make build"
```

Comment:
I think the source directory has to be set, the command gets run from root. Not sure if this file also has to be in the root.

Reply:
It looks like the Netlify config allows for a non-root directory, which seems better to me than having the website config in the root.